lib/knife-art/version.rb

```diff
@@ -1,3 +1,3 @@
 module KnifeArt
-  VERSION = '1.0.0'
+  VERSION = '1.0.1'
```

Comment:
Totally happy to remove this from the changeset if y'all have another process for version bumping.

Reply:
Did this so I could make a tag and release and "publish" a gem for testing. https://github.com/robbkidd/knife-art/releases/tag/1.0.1